1.0.0 upgrade

benbeltzer opened the pull request:
### What does this pr do?
- bumps version to 1.0.0
- removes `environment` param
- removes use of authorization code in `onComplete`
- removes `clientId, template, phone, email` params
- added `showCloseModalButton` param, which allows for differentiation between modal with and without close button

### How should this be tested?
- Use [this](https://github.com/berbix/onboarding-sample-integrations/pull/6) new sample app. See README of that app for instructions on how to do so

horaceko commented on src/index.jsx:
```diff
@@ -89,56 +94,32 @@ class BerbixVerify extends React.Component {
     }
-    const token = clientToken || continuation;
-    const template = templateKey || role;
+    const token = clientToken;
```
You can probably just use `clientToken` directly here.

benbeltzer replied:
lollll i wrote the same comment on your PR!

horaceko commented on src/index.jsx:
```diff
@@ -89,56 +94,32 @@ class BerbixVerify extends React.Component {
-    const token = clientToken || continuation;
-    const template = templateKey || role;
+    const token = clientToken;
     var options = ["sdk=BerbixReact-" + SDK_VERSION];
```
Let's switch these over to use template strings, for consistency.

benbeltzer replied:
agreed

horaceko reviewed (commented):
Generally looks good! Though, did you mean to nuke the `package-lock.json` file?

benbeltzer commented:
@horaceko addressed your feedback!

re:
> did you mean to nuke the package-lock.json file?

I didn't exactly nuke it. When I reran yarn install, it got rid of this block, which we spoke about during the mini-sprint. Do you remember that?

```
"packages": {
    "": {
```

horaceko commented:
@benbeltzer Aha, yes, that's right. Looks great then!
